src/platform/arm-nuttx/iotjs_module_gpio-arm-nuttx-stm32f.inl.h

```diff
@@ -0,0 +1,445 @@
+/* Copyright 2016 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef IOTJS_MODULE_GPIO_ARM_NUTTX_STM32F_INL_H
```

Comment:
Is there any specific reason that you named is as *stm32f*, not *stm32*? I didn't check every details, but it seems that "stm32_gpio.h" file also covers stm32l\*, not only stm32f\*